index.js

```diff
@@ -0,0 +1,124 @@
+$(document).ready(() => {
+  const baseUrl = "https://trektravel.herokuapp.com";
+
+  const travelList = function travelList() {
+    const listPartial = "/trips";
+    if (".individual-trip-section") {
+      $(".individual-trip-section").remove();
+    }
+    if ("#ajax-results p") {
+      $("#ajax-results p").remove();
+    }
+    $.get(baseUrl + listPartial, response => {
+      $(".list-trips").html(
+        `<div class = "removable-trip-list  row small-up-2 medium-up-2 large-up-3 "></div>`
+      );
+      for (let trip of response) {
+        $(".removable-trip-list").append(
+          `<div class = "container column column-block "><h3 class="trip" > ${
+            trip["name"]
+          }</h3> <div class="overlay"  id ="${trip["id"]}">
+    <div class="text">Learn More</div>
+  </div><div>`
+        );
+      }
+    })
+      .fail(function(response) {
+        console.log(response);
+        $("#fail").html("<p>Request was unsuccessful</p>");
+      })
+      .always(function() {
+        console.log("always even if we have success or failure");
+      });
```

Comment:
You probably don't need `always` callbacks on all of these. I think of `always` like the `ensure` clause of a `begin/rescue` block: every once in a very long while it's incredibly useful, and otherwise I forget it's even there.